src/lib/protocols/basicmessage/BasicMessageService.ts

```diff
@@ -36,7 +37,7 @@ class BasicMessageService {
       tags: { from: connection.theirDid || '', to: connection.did || '' },
     });
     await this.basicMessageRepository.save(basicMessageRecord);
-    connection.emit('basicMessageReceived', message);
+    this.emit('newMessage', { verkey: connection.verkey, message });
```

Comment:
I think it would be better to make this a constant or enum, so subscribing is not prone to typos. This way it also becomes easier to add typing to events (i.e. subscribe to 'newMessage' type always return a verkey and message when called)

Reply:
Good point. I thought about it. I wasn't sure if it's necessary to solve it now. There is a question of a naming convention of such event types:
- Tense: `stateChange`, `stateUpdate` vs. `stateChanged`, `stateUpdated`, I picked the latter, according to the CQRS naming convention, but I see a lot of examples of the first option in the JS world.
- Case: `stateChanged`, `state-changed`, `StateChanged`, `statechange` ordered by my preferences :)

Reply:
Nice. I would prefer `stateChanged` and `stateUpdated` over `stateChange` and `stateUpdate`. And i would prefer camelCase for naming (`stateChanged`). Not based on any valuable arguments, just preference :)

Reply:
Then we're on the same page :)